packages/fx-core/src/plugins/resource/aad/graph.ts

```diff
@@ -111,6 +111,34 @@ export namespace GraphClient {
     );
   }
 
+  export async function checkPermission(
+    graphToken: string,
+    objectId: string,
+    userObjectId: string
+  ): Promise<boolean> {
+    if (!objectId) {
+      throw new Error(
+        `${GraphClientErrorMessage.CheckPermissionFailed}: ${GraphClientErrorMessage.AppObjectIdIsNull}.`
+      );
+    }
+
+    if (!userObjectId) {
+      throw new Error(
+        `${GraphClientErrorMessage.CheckPermissionFailed}: ${GraphClientErrorMessage.UserObjectIdIsNull}.`
+      );
+    }
+
+    const instance = initAxiosInstance(graphToken);
+    const response = await instance.get(
```

Comment:
Do we need to add try catch block in case of network error and other unknown errors?

Reply:
All the errors here will be caught in AadAppClient.ts and throw as user/system error. We don't need try catch here.

Reply:
OK